ocs_ci/ocs/ui/helpers_ui.py

```diff
@@ -186,3 +286,70 @@ def get_element_type(element_name):
     """
 
     return (f"//a[contains(@title,'{element_name}')]", By.XPATH)
+
+
+def verify_storage_class_ui(setup_ui, sc_name):
+    """
+    Test for verifying storage class details via UI
+
+    """
+    base_ui_obj = PageNavigator(setup_ui)
+
+    ocp_version = get_ocp_version()
+    pvc_loc = locators[ocp_version]["storage_class"]
+
+    base_ui_obj.refresh_page()
+    base_ui_obj.navigate_storageclasses_page()
+    logger.info("Click on Dropdown and Select Name")
+    base_ui_obj.do_click(pvc_loc["sc-dropdown"])
+    base_ui_obj.do_click(pvc_loc["name-from-dropdown"])
+    logger.info("Search Storage Class with Name")
+    base_ui_obj.do_send_keys(pvc_loc["sc-search"], text=sc_name)
+    logger.info("Click and Select Storage Class")
+    base_ui_obj.do_click(format_locator(pvc_loc["select-sc"], sc_name))
+    # Verifying Storage Class Details via UI
+    logger.info("Verifying Storage Class Details via UI")
+    sc_name = base_ui_obj.check_element_text(expected_text=sc_name)
+    if sc_name:
+        logger.info(f"Storage Class '{sc_name}' Found")
+        return True
+    else:
+        logger.error(f"Storage Class '{sc_name}' Not Found, Verification Failed")
+        return False
+
+
+def delete_storage_class_ui(setup_ui, sc_name):
```

Comment:
Why did you add `delete_storage_class_ui` on 'helpers_ui.py' file?

Reply:
because create_storage_class_ui and delete_storage_class_ui are helper functions and will be used in a lot of PR's going forward.